docs/source/developer/developer.md

```diff
@@ -148,6 +148,11 @@ At present, the following query types are accessible through FlowAPI:
 
     Count of events (optionally of specific types) for individual subscribers in a time period.
 
+- `pareto_interactions`
+
+    Returns a two columns, a subscriber, pareto - the proportion
+    of that subscribers contacts who account for the requested
+    proportion (0.8, by default) of their interactions in this time period.
```

Comment:
While this is a copy of the docstring in `ParetoInteractions`, I suggest brushing up the grammar and phrasing a little (also in the original docstring):
```
    Returns two columns: `subscriber`, `pareto` - where the latter is the
    proportion of that subscriber's contacts who account for the requested
    proportion (0.8, by default) of their interactions in this time period.
```